akka-cluster-sharding/src/main/resources/reference.conf

```diff
@@ -94,6 +94,10 @@ akka.cluster.sharding {
   # works only for state-store-mode = "ddata"
   updating-state-timeout = 5 s
 
+  # The interval beteween which remembered entities are restarted when initialing the shard
```

Comment:
This section is a bit terse. Can you please expand this and explain what exactly will happen if set for a particular non-zero value?

Also, I would probably prefer the setting to be either "off" or a positive number. That means that the name must be somewhat changed so that "off" makes sense.